src/java/arjdbc/jdbc/RubyJdbcConnection.java

```diff
@@ -1677,7 +2161,7 @@ protected IRubyObject mapToResult(final ThreadContext context, final Ruby runtim
     }
 
     @Deprecated
-    protected IRubyObject jdbcToRuby(final Ruby runtime,
+    protected final IRubyObject jdbcToRuby(final Ruby runtime,
```

Comment:
I am not a huge fan of final as a keyword in general but I really dislike final methods.  I would like the option of a subclass being able to override this method.

Reply:
yep, thx - here it makes no sense really - will remove.
but "internal" (sensitive) pieces I would like to keep final